src/main/java/net/sf/jabref/logic/CustomEntryTypesManager.java

```diff
@@ -1,30 +1,28 @@
 package net.sf.jabref.logic;
 
-import java.util.ArrayList;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Optional;
+import java.util.stream.Collectors;
 
 import net.sf.jabref.model.EntryTypes;
 import net.sf.jabref.model.database.BibDatabaseMode;
 import net.sf.jabref.model.entry.CustomEntryType;
-import net.sf.jabref.model.entry.EntryType;
 import net.sf.jabref.preferences.JabRefPreferences;
 
 public class CustomEntryTypesManager {
 
-    public static final List<EntryType> ALL = new ArrayList<>();
     /**
      * Load all custom entry types from preferences. This method is
      * called from JabRef when the program starts.
      */
     public static void loadCustomEntryTypes(JabRefPreferences prefs) {
```

Comment:
Not entirely, unfortunately. See: https://github.com/JabRef/jabref/wiki/High-Level-Documentation
The CLI is not supposed to call the gui, so anything accessed from GUI and CLI should be in logic, but it should be independent of the preferences. You could move the save and leave the load for instance.